include/wx/math.h

```diff
@@ -164,9 +164,21 @@ inline int wxRound(float x)
 
 inline int wxRound(long double x) { return wxRound(double(x)); }
 
+// For compatibility purposes, define wxRound() overloads for integer types
+// too, as this used to compile with wx 3.0.
+#if WXWIN_COMPATIBILITY_3_0
+
 wxDEPRECATED_MSG("rounding an integer is useless")
 inline int wxRound(int x) { return x; }
 
+wxDEPRECATED_MSG("rounding an integer is useless")
+inline int wxRound(short x) { return x; }
+
+wxDEPRECATED_MSG("rounding an integer is useless")
+inline int wxRound(long x) { return x; }
```

Comment:
```suggestion
inline int wxRound(long x) { return int(x); }
```

Reply:
This is actually bothersome now that I think about it because previously you'd get an assert when using a long out of int range (because it would be converted to double), while now it will be silent. But I don't see any simple way to fix this other than
```suggestion
inline int wxRound(long x) { return wxRound(static_cast<double>(x)); }
```
which looks ridiculuous.